pkg/provisioners/gcppubsub/util/testcreds/helper.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testcreds
```

Comment:
Optional: Why is this a separate package from the actual test code?

Reply:
I didn't want to mix test code in with production code. If put into the same package as `creds.go`, then all the real code would also have these fake functions already imported. 

It also makes for nice instantiation, `testcreds.MakeSecretWithCreds()`.